packages/spectacular/README.md

```diff
@@ -4,43 +4,65 @@
-| ---------------------------------------- | --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
-| `createFeatureHarness`                   | Factory for `SpectacularFeatureHarness`       | Configures `SpectacularFeatureTestingModule`, bootstraps `SpectacularAppComponent` and navigates to the default feature route. |
-| `CreateFeatureHarnessOptions`            | Options for `createFeatureHarness`            | Feature harness options.                                                                                                       |
-| `SpectacularFeatureHarness`              | Interface                                     | Feature harness data structure.                                                                                                |
-| `SpectacularFeatureTestingModuleOptions` | Options for `SpectacularFeatureTestingModule` | Feature testing options for `SpectacularFeatureTestingModule.withFeature`.                                                     |
-| `SpectacularFeatureTestingModule`        | Angular testing configuration module          | Configures the `RouterTestingModule` and provides Spectactular services for testing feature modules.                           |
-| `SpectacularFeatureLocation`             | Service                                       | A subset of Angular's `Location` API adjusted to the Angular feature module under test.                                        |
-| `SpectacularFeatureRouter`               | Service                                       | A subset og Angular's `Router` API adjusted to the Angular feature module under test.                                          |
+| Export name                              | Kind                                          | Description                                                                                                                 |
+| ---------------------------------------- | --------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| `createFeatureHarness`                   | Factory for `SpectacularFeatureHarness`       | Configure `SpectacularFeatureTestingModule`, bootstrap `SpectacularAppComponent` and navigate to the default feature route. |
+| `CreateFeatureHarnessOptions`            | Options for `createFeatureHarness`            | Feature harness options.                                                                                                    |
+| `SpectacularFeatureHarness`              | Interface                                     | A harness for testing an Angular feature module.                                                                            |
```

Comment:
Maybe something like this?
```suggestion
| `SpectacularFeatureHarness`              | Interface                                     | A harness for testing an Angular feature module routing.                                                                            |
```

Reply:
There's routing modules and feature modules. Not feature routing modules.